### DISCUSSION TITLE
Incremental Update?

### DISCUSSION THREAD

**Discussion by eeriemyxi** [sigma67/spotify_to_ytmusic#123] [Q&A]
Hi, I was trying out this tool and am liking it so far. I ran `spotify_to_ytmusic liked` and it did work as expected (although you should probably deprecate method 2 from `setup` since it doesn't work anymore). I was expecting it to just do an incremental update of the playlist upon running it a second time but seems like it's not supported yet since when I ran `spotify_to_ytmusic liked` it just started fetching the whole playlist again.

Am I missing something or is somebody yet to implement it?

**Comment by sigma67** [comment]
Not supported yet. PR welcome. Someone tried something similar for `all` in https://github.com/sigma67/spotify_to_ytmusic/pull/84/files
